src/cpp/session/modules/connections/SessionConnections.cpp

```diff
@@ -74,7 +76,7 @@ SEXP rs_connectionOpened(SEXP connectionSEXP)
    if (!r::sexp::isNull(actionList))
    {
       std::vector<std::string> actionNames;
-      error = r::sexp::getNames(actionList, &actionNames);
+      r::sexp::getNames(actionList, &actionNames);
```

Comment:
It does, thanks for catching that! https://github.com/rstudio/rstudio/commit/4246a44ca2c94281cebfb207fcb9f4b5b718a1f1.